src/main/java/com/google/sps/data/Recipe.java

```diff
@@ -23,16 +23,18 @@ public Recipe(long id, String name, String imageBlobKey, String description) {
     this.name = name;
     this.imageBlobKey = imageBlobKey;
     this.description = description;
+    authorID = "";
     ingredients = new ArrayList<String>();
     steps = new ArrayList<String>();
     tags = new ArrayList<String>();
     published = false;
   }
 
-  public Recipe(long id, String name, String imageBlobKey, String description, ArrayList<String> tags, 
+  public Recipe(long id, String name, String authorID, String imageBlobKey, String description, ArrayList<String> tags, 
       ArrayList<String> ingredients, ArrayList<String> steps, boolean published, long popularity) {
     this.id = id;
     this.name = name;
+    this.authorID = authorID;
```

Comment:
Nit: are we not using the convention on `authorId` instead of `authorID`?

Reply:
the methods/functions we keep in camelcase (i.e. getAuthorId, getId) but the variable names we said could stay as ID
![image](https://user-images.githubusercontent.com/51170036/88421229-2f540980-cdb6-11ea-8292-74cef6c3d68f.png)